minesweeper.py

```diff
@@ -279,7 +279,16 @@ def highscore(mins, secs):
             top_10_times.insert(index, converted_to_seconds)  # SAME AS ABOVE
 
 
+def save_json_file():
+    json_dict["height_slider"] = int(height_slider.scale.get())
+    json_dict["width_slider"] = int(width_slider.scale.get())
+    json_dict["difficulty_slider"] = int(difficulty_slider.scale.get())
+    with open("game_data.json", "w") as game_info:
```

Comment:
Couple things: variable name, always putting file to `where_this_file_is`.

```suggestion
    with open(where_this_file_is / "game_data.json", "w") as file:
```

Also change `json.dump` to use the new `file` variable name, and change the other code to use `where_this_file_is / "game_data.json"`